src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,59 @@
+import cn from 'classnames';
+import { useDispatch, useSelector } from 'react-redux';
+import { RootState } from '../../app/store';
+import { actions } from '../../features/currentTodo';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo,
+};
+
+export const TodoItem: React.FC<Props> = ({ todo }) => {
+  const { id, completed, title } = todo;
+  const currentTodoId = useSelector<RootState>(state => state.currentTodo?.id);
+  const dispatch = useDispatch();
+  const isTodoSelected = currentTodoId === id;
+
+  return (
+    <tr
+      data-cy="todo"
+      className={cn({ 'has-background-info-light': isTodoSelected })}
```

Comment:
Don't cut variable name. Use meaningful naming
```suggestion
      className={classnames({ 'has-background-info-light': isTodoSelected })}
```